### PR TITLE
feat: Contact page

### LINKED ISSUE #91: Change photo of Anna
Please, exchange the existing photo with this one: 

Photo

### LINKED ISSUE #105: Contact page
Add content to the Contact page

### PR DESCRIPTION
**Pull request by dkostenko-bjerk**
Close #105
Close #91

### CODE REVIEW

File: web/src/components/contact/employee-list-item.tsx [hunk 1]
```diff
@@ -1,40 +1,34 @@
 import { Box, Container, Flex, Link, Text } from '@theme-ui/components';
-import Img, { FluidObject } from 'gatsby-image';
 import React from 'react';
 
 export interface EmployeeListItemProps {
-  photo?: FluidObject;
+  photo?: any;
```
Comment (cobraz) [inline web/src/components/contact/employee-list-item.tsx:5]: Why `any`? We have activated `@typescript-eslint/no-explicit-any`.

File: web/src/pages/contact.tsx [hunk 2]
```diff
@@ -1,20 +1,43 @@
 import { Box, Flex, Heading, Label, Link } from '@theme-ui/components';
 import React from 'react';
 import { SystemStyleObject } from 'theme-ui';
-import MapSvg from '../assets/map.svg';
 import { EmployeeList } from '../components/contact/exployee-list';
 import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 
+const empoyeeList = [
+  {
```
Comment (cobraz) [inline web/src/pages/contact.tsx:9]: I talk about this in #108 as well.

It is relevant here as well, IMHO. 

re https://github.com/bjerkio/website/pull/108#discussion_r600901491

**Review by cobraz — APPROVED** [review]
LGTM 🎉

**Review by AnneMatilde — CHANGES_REQUESTED** [review]
The text has the wrong size and weight – this is throughout the subpages. 
This is how it looks now:
image

And this is how it should look:
image

Picture of Niklas and Anna is blury.

Header is missing background on scroll.

Text underneath the pictures is also too small and squeezed:
image

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde where can I get not blurry images?

**Comment by cobraz** [comment]
> @AnneMatilde where can I get not blurry images?

Isn't the designs made in Adobe XD or Figma? Should be possible to export the assets directly.

/cc @AnneMatilde

**Comment by dkostenko-bjerk** [comment]
@cobraz They are blurry there too

@seacurrent @AnnaBjerk Do you have non-blurry photos?

**Comment by AnneMatilde** [comment]
The design is in Adobe XD. The images are blury to begin with – so we need better images.

**Comment by seacurrent** [comment]
BNS
You could probably use this profile pic.

**Comment by cobraz** [comment]
This is actually a nice test of our user experience, maybe everyone can upload their own pictures?

re #91

**Comment by AnnaBjerk** [comment]
> The text has the wrong size and weight – this is throughout the subpages.
> This is how it looks now:
> image
> 
> And this is how it should look:
> image
> 
> Picture of Niklas and Anna is blury.
> 
> Header is missing background on scroll.
> 
> Text underneath the pictures is also too small and squeezed:
> image

I have made a separate issue on the disappearing background in header. I'll make an issue on text as well. As I understood the text is easier to fix after the pages have been merged. Is this correct, @dkostenko-bjerk ?

**Comment by dkostenko-bjerk** [comment]
@AnnaBjerk Yes

**Comment by AnnaBjerk** [comment]
@AnneMatilde Could you aprove this for now and I'll make an issue on text using the photos you posted so that we can keep issues separated and in smaller pieces 😸

**Review by AnnaBjerk — APPROVED** [review]

**Comment by AnnaBjerk** [comment]
> They are blurry there too
> 
> @AnnaBjerk Do you have non-blurry photos?

I'll try to find the bigger original. If that helps?

**Comment by cobraz** [comment]
Images can be replaced later, we don't need to wait for that :)

**Comment by AnnaBjerk** [comment]
> Images can be replaced later, we don't need to wait for that :)

Agreed. Let's land this!

**Comment by AnnaBjerk** [comment]
> They are blurry there too
> 
> @AnnaBjerk Do you have non-blurry photos?

20191118_port-f

/cc: @dkostenko-bjerk

**Pull request merged**